specification/sql/resource-manager/readme.md

```diff
@@ -693,6 +694,120 @@ override-info:
 
 The following packages are each composed of all apis from only one api-version.
 
+### Tag: package-preview-2020-11
```

Comment:
can you remove this section? we already have 2020-11 tag above, right?

Reply:
I removed it from L35 in a prior commit